apply/apply.c

```diff
@@ -146,14 +146,16 @@ void main(int argc, char *argv[]) {
 	/* Declare the command string. */
 	cmd = strdup(arg[0]);
 	shift(arg, cmdc);
+	cmdlen = strlen(cmd);
 
 	/*
 	 * Initialize the magias[] array
 	 * with invalid magic numbers, so
 	 * we will avoid false-positives
 	 * for c=0.
 	 */
-	memset(magias, -1, (10 * (sizeof(uint8_t))));
+	magias = calloc(cmdlen, sizeof(uint8_t));
+	memset(magias, -1, cmdlen);
```

Comment:
Wrong usage of memset, wouldn't work if `magias` wasn't an `uint8_t`.

Reply:
Also, -1 is invalid for `uint8_t`, you probably mean 255.

Reply:
> Wrong usage of memset, wouldn't work if `magias` wasn't an `uint8_t`.

How can I fix this too?